plaid-stl/src/datetime.rs

```diff
@@ -1,18 +1,64 @@
-use chrono::{DateTime, Utc};
+use chrono::{NaiveDate, Utc, DateTime};
 use serde::{Deserialize, Deserializer};
 
-// Custom deserializer for an optional DateTime<Utc>
-pub fn deserialize_option_timestamp<'de, D>(deserializer: D) -> Result<Option<DateTime<Utc>>, D::Error>
+// Custom deserializer from a ISO 8601 string to a DateTime<Utc>
+pub fn deserialize_rfc3339_timestamp<'de, D>(deserializer: D) -> Result<DateTime<Utc>, D::Error>
```

Comment:
Interesting choice in the comment to refer to it as ISO 8601 and the function RFC3339?

Reply:
:P I want to call it ISO 8601 but the chrono function uses RFC 3339